amp/system-oracle/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM registry.redhat.io/3scale-amp2/system-rhel7:3scale2.11
 
 USER root
 
-COPY ./oracle-client-files/instantclient-basic*-linux.x64*.zip \
```

Comment:
`linux`  prefix will also work for `ppc64le`?

Reply:
How do ppc64le oracle client files look like?

Reply:
The instantclient [package](https://download.oracle.com/otn_software/linux/instantclient/193/instantclient-basic-linux.leppc64.c64-19.3.0.0.0dbru.zip)  has *linux* prefix in it, so does SDK,odbc packages.